doc/stdenv/meta.chapter.md

```diff
@@ -86,6 +86,17 @@ meta.platforms = lib.platforms.linux;
 
 Attribute Set `lib.platforms` defines [various common lists](https://github.com/NixOS/nixpkgs/blob/master/lib/systems/doubles.nix) of platforms types.
 
+### `badPlatforms` {#var-meta-badPlatforms}
+
+The list of Nix platform types on which the package is known not to be buildable.  Hydra will never create prebuilt binaries for these platform types, even if they are in `meta.platforms`.  In general it is preferable to set `meta.platforms = lib.platforms.all` and then exclude any platforms on which the package is known not to build.  For example, a package which requires dynamic linking and cannot be linked statically could use this:
```

Comment:
Would be great if we had a link to the relevant terms here:
- platform types
- Hydra
- `meta.platforms`

Reply:
2b6de0562a4499a053e212ae062c10de96d4854c